modin/core/execution/dispatching/factories/factories.py

```diff
@@ -509,7 +509,11 @@ def _read_sql(cls, **kwargs):
         params=_doc_io_method_kwargs_params,
     )
     def _read_csv_glob(cls, **kwargs):
-        supported_execution = ("ExperimentalPandasOnRay", "ExperimentalPandasOnUnidist")
+        supported_execution = (
+            "ExperimentalPandasOnRay",
+            "ExperimentalPandasOnUnidist",
+            "ExperimentalPandasOnDask",
+        )
```

Comment:
could you extract it to a module-level constant?.. it's been used multiple times here